aries_vcx/src/core/profile/modular_wallet_profile.rs

```diff
@@ -0,0 +1,51 @@
+use std::sync::Arc;
+
+use crate::{
+    error::VcxResult,
+    plugins::{
+        anoncreds::{base_anoncreds::BaseAnonCreds, credx_anoncreds::IndyCredxAnonCreds},
+        ledger::{
+            base_ledger::BaseLedger,
+            indy_vdr_ledger::{IndyVdrLedger, IndyVdrLedgerPool},
+        },
+        wallet::base_wallet::BaseWallet,
+    },
+};
+
+use super::profile::Profile;
+
+pub struct LedgerPoolConfig {
+    pub genesis_file_path: String,
+}
+
+#[allow(dead_code)]
+#[derive(Debug)]
+pub struct ModularWalletProfile {
```

Comment:
Any other naming suggestions for this? The intention is for this to be a profile implementation which is constructed using our new modular deps (indy-vdr + indy-credx) + any `dyn BaseWallet` which you plug in - (i.e. a modular wallet).

Reply:
`ModularWalletProfile` is okay